packages/edit-site/src/components/editor/utils.js

```diff
@@ -103,13 +103,28 @@ function getPresetMetadataFromStyleProperty( styleProperty ) {
 export const LINK_COLOR = '--wp--style--color--link';
 export const LINK_COLOR_DECLARATION = `a { color: var(${ LINK_COLOR }, #00e); }`;
 
+const filterColorsFromCoreOrigin = ( path, setting ) => {
```

Comment:
Will this get out of sync with the implementation in block editor?

Reply:
I feel ultimately here, we'd need a way to get both

```
useSetting( 'color.palette' ) // return theme colors
useSetting( 'color.defaultPalette' ) // return default palette 
```

and same for other settings. It's not important now, but we should make sure it's something we can achieve later.